Project.toml

```diff
@@ -5,6 +5,7 @@ version = "2.2.0"
 
 [deps]
 DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
+DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
```

Comment:
This should not be necessary. We should use OrdinaryDiffEq only for now.

Reply:
@janlisse : I needed the package to access ODEFunction [here](https://github.com/JuliaEnergy/PowerDynamics.jl/pull/54/files#diff-d3c102d899631c26c9a6dae5a01633f4R16) and extract the mass matrix. Do you see an alternative here?

Reply:
Yes, you can just use ```using OrdinaryDiffEq: ODEFunction```

Reply:
ok, done :)